onboarding/source/controller.c

```diff
@@ -97,16 +116,29 @@ static void controllerTask(void * pvParameters) {
     obc_error_code_t lightServiceStatus = initLightService();
     if (lightServiceStatus != OBC_ERR_CODE_SUCCESS) {
         /* USER CODE BEGIN */
-        // Deal with error when initializing light service task and/or queue
+        char str[] = "Error: Light service was not properly initialized";
```

Comment:
No need to declare an array for the string. You can just pass `"Error: Light service was not properly initialized"` into `sciPrintf`